### PR TITLE
Add/Remove widgets

### PR DESCRIPTION
**Pull request by massimocode**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,8 +16,9 @@
         <div class="row">
           <!-- ko foreach: calculators -->
           <div class="col-lg-6 col-md-4 col-sm-6 col-xs-12 calculator">
```
Comment (massimocode) [inline index.html]: `Each widget would be the same size as well.` from https://github.com/SMH110/widgets/issues/44

It looks to me as though the To Do List and the Calculator would be different sizes at some resolutions

File: index.html [hunk 2]
```diff
@@ -29,9 +30,16 @@
...
+
+
+        <!-- ko foreach: toDoLists -->
+        <div class="row toDoList">
+          <!-- ko component: 'toDoList' -->
+          <!-- /ko -->
+        </div>
+        <button class="btn btn-block btn-danger toDoList-remove" data-bind="click: $parent.removeToDoList">Remove</button>
         <!-- /ko -->
+        <button  id="remove-todo" class="btn btn-block btn-primary" data-bind="click: addToDoList">Add To-Do-List</button>
```
Comment (massimocode) [inline index.html]: What is the ID on this button for?

File: lib/app.js [hunk 3]
```diff
@@ -9,7 +9,8 @@
 } (function (ko, Calculator, ToDoList) {
 
   function App() {
-    let calculators = ko.observableArray([new Calculator()]);
+    let calculators = ko.observableArray();
+
 
     function addCalculator() {
       calculators.push(new Calculator());
```
Comment (massimocode) [inline lib/app.js]: I am wondering, why do you need to add `new Calculator()` for? When is that viewmodel ever used or bound to?

File: index.html [hunk 4]
```diff
@@ -29,9 +30,16 @@
       </div>
 
       <div class="col-lg-6 col-md-12">
```
Comment (massimocode) [inline index.html]: I am wondering, how can I put a To Do list before a calculator? i.e. if I want the todo list on the left, it doesn't look like that will be possible

**Comment by massimocode** [comment]
The todo list is still double the size of the calculator. I can't upload images here at the moment but I will email you a screenshot

File: lib/app.js [hunk 5]
```diff
@@ -13,7 +13,7 @@
 
 
     function addCalculator() {
-      calculators.push(new Calculator());
+      calculators.push(" :) ");
```
Comment (massimocode) [inline lib/app.js]: With this change, I can't remove calculators / todo lists any more. Why do you think that is?

Comment (SMH110) [inline lib/app.js]: I really don't know why?

File: index.html (hunk 4 above)
Comment (SMH110) [inline index.html]: it is possible https://github.com/SMH110/widgets/pull/52/commits/21fa303195e1b42545fb519e4086f258be202e1b

**Comment by massimocode** [comment]
I've noticed you've moved the to do lists to the left. What is the user wants them on the right? The comment I put earlier said `I am wondering, how can I put a To Do list before a calculator? i.e. if I want the todo list on the left, it doesn't look like that will be possible`. It's not a requirement to be on the left. It's a requirement for the user to be able to have them in any order

File: lib/app.js (hunk 5 above)
Comment (SMH110) [inline lib/app.js]: now it's working
https://github.com/SMH110/widgets/commit/4c008ecdac1ccfc5387a2c59ce91d0373828651b

File: index.html (hunk 4 above)
Comment (SMH110) [inline index.html]: I don't think it is possible

**Pull request merged**